src/hooks/useUsers/useUsers.js

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from 'react';
+import { useQuery } from 'react-query';
+
+import { useNamespace, useOkapiKy } from '@folio/stripes/core';
+
+export const useUsers = ({ tenant = '' }) => {
+  const [namespaceKey] = useNamespace({ key: 'relatedUsers' });
+  const ky = useOkapiKy({ tenant });
+  const [users, setUsers] = useState([]);
+
+  const { data, isFetching, isSuccess } = useQuery({
+      queryKey: [namespaceKey, tenant],
+      queryFn: () => ky.get('users').json(),
+    });
+
+  useEffect(() => {
+    if (isSuccess && data?.users) {
+      setUsers(data.users);
+    }
+  }, [data, isSuccess]);
```

Comment:
I don't quite understand why use `useEffect` here, instead of returning `data` or the default result in case of failure, could you please add more details?

Reply:
I used the same logic is done https://github.com/folio-org/stripes-authorization-components/blob/master/lib/hooks/useAuthorizationRoles/useAuthorizationRoles.js

Reply:
It seems that part of the code was just copied from `ui-authorization-roles` as is. But this effect hook is really strange for me, and I see no reason to use it unless I missed something.